modules/shared-resources/key.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_kms_key" "encryption" {
```

Comment:
Like we chatted about in person, at some point it would be good to think about whether some encrypted data is more sensitive (e.g. the S3 buckets that contain uploaded files) and need separate keys.